delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/projects/ProjectCountsCollection.scala

```diff
@@ -45,16 +48,21 @@ object ProjectCountsCollection {
   /**
     * The counts for a single project
     *
-    * @param value  the number of events existing on the project
+    * @param eventsCount                the number of events existing on the project
+    * @param resourcesCount             the number of resources existing on the project
     * @param lastProcessedEventDateTime the time when the last count entry was created
     */
-  final case class ProjectCount(value: Long, lastProcessedEventDateTime: Instant)
+  final case class ProjectCount(eventsCount: Long, resourcesCount: Long, lastProcessedEventDateTime: Instant)
```

Comment:
We could drop the count on the values as we have it in the class name

Reply:
I kept the count on the name because it is easier for circe serialization/deserialization. Otherwise I need a `Configuration` serializer and so on and I don't think it is worth it.